reference/api/keys.md

```diff
@@ -182,80 +237,84 @@ Get information on the specified key. Attempting to use this endpoint with a non
 }
 ```
 
-For an explanation of these fields, see the [get all keys endpoint](#returned-fields).
+For an explanation of these fields, see the [key object](#key-object).
 
 ## Create a key
 
 <RouteHighlighter method="POST" route="/keys"/>
 
 Create an API key with the provided description, permissions, and expiration date.
 
-Only the `indexes`, `actions`, and `expiresAt` fields are mandatory.
-
 ### Body
```

Comment:
I think we should do a table for the body, with links to the extended object definition where necessary. Just seems cleaner, less redundant, and more consistent w/ what we're doing elsewhere. WDYT?